.codeName

```diff
@@ -0,0 +1 @@
+Koi
```

Comment:
Please don't use camelCase filenames. Also, does this really have to be in the root directory of the repository?

Reply:
how about something like this https://github.com/NixOS/nixpkgs/pull/48588 ?